### ISSUE TITLE
Weird error: Cannot move PDF

### ISSUE THREAD

**Issue by maxheld83** [LaTeXing/LaTeXing#188]
Sometimes, LaTeXing (though not RStudio on the same file, or CLI equivalents), will randomly give this pop-up error

```
 "Could not move the pdf or synctex file. Close the pdf viewer to reset the file access or other permission problems."
```

The only way to fix it is to delete the output dir, as well as all output and intermediary files.

This seems to affect `*.Rnw` files.

I'll try and track down the conditions under which this happens in greater detail.

**Comment by hbahamonde** [comment]
Hey there, I can confirm @maxheld83. I use two machines, one at work, the other one at home. Both sync via a small Sinology server. Just run the synced  `.rnw` copy at home, and it did not compile. After deleting everything but the  `.rnw`, it compiles fine.

**Comment by csch0** [comment]
Yes that just means that python is not allowed to remove the file. It could be because its still open, locked, or you user is not allowed to. Not sure from here, but basically thats the only meaning of this error.
